src/nft/components/explore/Cells/Cells.tsx

```diff
@@ -19,48 +69,85 @@ interface CellProps {
 
 export const CollectionTitleCell = ({ value }: CellProps) => {
   return (
-    <Row as="span" style={{ marginLeft: '52px' }}>
-      <img className={styles.logo} src={value.logo} alt={`${value.name} logo`} height={44} width={44} />
-      <span className={styles.title}>{value.name}</span>
+    <CollectionNameContainer>
+      <RoundedImage src={value.logo} />
+      <CollectionName>
+        <TruncatedSubHeader>{value.name}</TruncatedSubHeader>
+      </CollectionName>
       {value.isVerified && (
         <span className={styles.verifiedBadge}>
           <VerifiedIcon />
         </span>
       )}
-    </Row>
+    </CollectionNameContainer>
   )
 }
 
-export const WithCommaCell = ({ value }: CellProps) => <span>{value.value ? putCommas(value.value) : '-'}</span>
-
-export const EthCell = ({ value }: { value: number }) => (
-  <Row justifyContent="flex-end" color="textPrimary">
-    {value ? <>{formatWeiToDecimal(value.toString(), true)} ETH</> : '-'}
-  </Row>
+export const DiscreteNumberCell = ({ value }: CellProps) => (
+  <span>{value.value ? volumeFormatter(value.value) : '-'}</span>
 )
 
-export const VolumeCell = ({ value }: CellProps) => (
-  <Row justifyContent="flex-end" color="textPrimary">
-    {value.value ? <>{ethNumberStandardFormatter(value.value.toString())} ETH</> : '-'}
-  </Row>
-)
+export const EthCell = ({
+  value,
+  denomination,
+  usdPrice,
+}: {
+  value?: number
+  denomination: Denomination
+  usdPrice?: number
+}) => {
+  const denominatedValue =
+    denomination === Denomination.ETH
+      ? value
+      : usdPrice && value
+      ? usdPrice * parseFloat(formatEther(value))
+      : undefined
+  const formattedValue = denominatedValue
```

Comment:
should we use the formatting we have in the rest of the app?

Reply:
no they requested special formatting for the different cases here